### DISCUSSION TITLE
Interactsh Client doesn't work with BurpSuite

### DISCUSSION THREAD

**Discussion by byposeidon** [projectdiscovery/interactsh#327] [General]
Hello,

I want to use interactsh-client instead of burp collaborator for OOB interaction.
Example site with out-of-band vulnerability:  http://testphp.vulnweb.com/showimage.php?file=http://cb38eh9c67vbl4n0i9ngihge7nz1183yn.oast.online/?a

Burpsuite can't detect it as a ssrf vulnerability with interactsh. However, it can be detected with the collaborator.

What could be the cause of the error?
Screen: https://i.ibb.co/J7LdfcX/interactsh.png

**Comment by Mzack9999** [comment]
@byposeidon `interactsh-server` uses a different client-server protocol, which is incompatible with the burp-collaborator BIID-based one. If you want to use interactsh within Burp Suite, you should install the plugin at https://github.com/wdahlenburg/interactsh-collaborator

**Comment by noraj** [comment]
And could the be also possible to support Burp Collaborator protocol, so that one that has a interactsh server don't need a burp colalborator server. Because yes one can use the interctsh burp plugin for manual interraction but the burp scanner will still use burp collaborator and not interractsh. So if interractsh was compatible with burp protocol the burp scanner could also use interractsh.

**Reply by ehsandeep** [reply]
@noraj burp is a closed-source project, so it's not possible from the implementation point of view.

**Reply by noraj** [reply]
Right, I forgot about that. I thought the collaborator server was open source but it is not: https://portswigger.net/burp/documentation/collaborator/deploying

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
